.teamcity/_self/projects/WPComTests.kt

```diff
@@ -742,47 +742,46 @@ private object I18NTests : BuildType({
 		bashNodeScript {
 			name = "Run i18n tests"
 			scriptContent = """
-					shopt -s globstar
-					set -x
-
-					cd test/e2e
-					mkdir temp
+				shopt -s globstar
+				set -x
 
-					export URL=%URL%
-					export LIVEBRANCHES=false
-					export NODE_CONFIG_ENV=test
-					export TEST_VIDEO=true
-					export HIGHLIGHT_ELEMENT=true
-					export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL}\"}"
-					export TARGET=I18N
+				cd test/e2e
+				mkdir temp
 
-					# Instructs Magellan to not hide the output from individual `mocha` processes. This is required for
-					# mocha-teamcity-reporter to work.
-					export MAGELLANDEBUG=true
+				export NODE_CONFIG_ENV=test
+				export PLAYWRIGHT_BROWSERS_PATH=0
+				export TEAMCITY_VERSION=2021
+				export URL=%URL%
+				export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
+				export DEBUG=pw:api
+				export HEADLESS=true
+				export TARGET_DEVICE=desktop
 
-					# Decrypt config
-					openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
+				# Decrypt config
+				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
 
-					# Run the test
-					IFS="," read -r -a LOCALE <<< "%LOCALES%"
-					for locale in ${'$'}{LOCALE[@]}; do
-						BROWSERLOCALE="${'$'}{locale}" yarn magellan --config=magellan-i18n.json --max_workers=%E2E_WORKERS% --local_browser=chrome --mocha_args="--reporter mocha-multi-reporters --reporter-options configFile=mocha-reporter.json" || true
-					done
-				""".trimIndent()
-			dockerRunParameters = "-u %env.UID% --security-opt seccomp=.teamcity/docker-seccomp.json --shm-size=8gb"
+				# Run the test
+				IFS="," read -r -a LOCALE <<< "%LOCALES%"
+				for locale in ${'$'}{LOCALE[@]}; do
+					BROWSERLOCALE="${'$'}{locale}" yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%E2E_WORKERS% --group=i18n
```

Comment:
I think the env variable should `LOCALE` instead of `BROWSERLOCALE` - https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/browser-helper.ts#L36. I think that's likely the reason why @scinos got [zero tests running on this testing](https://github.com/Automattic/wp-calypso/pull/57934#issuecomment-965970094).